native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -423,7 +430,23 @@ void EventDispatcher::removeAllEventListeners() {
     hashListenerId = 1;
 }
 
-void EventDispatcher::dispatchCustomEvent(const CustomEvent &event) {
+void EventDispatcher::dispatchCustomEvent(const char *eventName, int argNum, ...) {
+    if (!eventName || !strcmp(eventName, "")) {
+        return;
+    }
+    CustomEvent event;
+    event.name = eventName;    
+    va_list vl;
+    va_start(vl, argNum);
+    // Step through the list.
+    for (int i = 0; i < argNum; ++i) {
```

Comment:
argNum is not needed, please refer the [doc](https://www.cplusplus.com/reference/cstdarg/va_end/#:~:text=Performs%20the%20appropriate%20actions%20to,been%20invoked%20from%20that%20function.)

Reply:
Yes, I can make it with while-loop

Reply:
Hard to end the loop, as all parameters can be translate to EventParameterType, so I finally keep the argNum..